HowManyBalloonsAlg.py

```diff
@@ -0,0 +1,15 @@
+"""How Many Balloons Alghorithm"""
+
+import math
+
+class BalloonCount(object):
+    def __init__(self, weight):
+        self.weight = weight
+
+    def how_many_balloons(self):
+        balloons = self.weight * 453.593 / 14
+        return math.ceil(balloons)
+
+weight = BalloonCount(int(input("Enter your weight ")))
```

Comment:
prefix any module level command with `if __name__ == '__main__':`
this prevents other modules from running this code everytime they import the module